static/app/views/performance/transactionSummary/transactionEvents/utils.tsx

```diff
@@ -150,3 +155,40 @@ export function mapShowTransactionToPercentile(
       return undefined;
   }
 }
+
+export function mapPercentileValues(percentileData?: TableDataRow | null) {
+  return {
+    p100: percentileData?.['p100()'],
+    p99: percentileData?.['p100()'],
```

Comment:
looks like this was a typo carried over from before. Can we check what is correct here?

```suggestion
    p99: percentileData?.['p99()'],
```